### ISSUE TITLE
Google Video streams?

### ISSUE THREAD

**Issue by BoredLama** [rleroi/stremio-anime#7]
I've been looking at some of the series that are missing streams in this add-on.

I've only seen the api reply with fb cdn streams, while on the actual anime site some don't have fb cdn streams at all, but they do have google video streams.

I'm uncertain of how you managed to discover their api as i couldn't find any reference whatsoever of it anywhere on the internet (and my google jitsu is strong), am presuming it required disassembling some phone app, although i couldn't find an official phone app for their site either.

Regardless, do you think it's possible to find an api endpoint that also serves google video streams? As those should work in Stremio as well as fb cdn ones.

**Comment by rleroi** [comment]
You're right, I'm using the same API as the KissAnime app. They used to advertise the app on their mobile website, but I don't see it anymore.
Some info about the API I've collected can be found in https://github.com/rleroi/stremio-anime/blob/master/doc/api.txt
I didn't see an alternative/parameter to get other sources, but I think Ive seen some videos coming from sources other than fbcdn in the addon. It just seems that most are coming from fbcdn.

**Comment by BoredLama** [comment]
I've seen the api docs u made, but I still feel like we're missing something here somehow..

Was it an android or a ios app? Do you still have it downloaded somewhere?

**Comment by rleroi** [comment]
I might have the app on my pc, but i think the app mirror sites on google could have it too. I will check later and send it to you if you want. It is Android.
Another strange thing about the API i found is that Ecchi genre does not work on the addon but it does work on the app. Other genres all seem to work. Also there are multiple urls defined in the app sourcecode, but i think the others are for ads etc. But maybe i've missed something.
